lib/librte_hash/rte_hash.h

```diff
@@ -419,6 +419,32 @@ rte_hash_lookup_bulk(const struct rte_hash *h, const void **keys,
  */
 int32_t
 rte_hash_iterate(const struct rte_hash *h, const void **key, void **data, uint32_t *next);
+
+/**
+ * Iterate over entries that conflict with a new entry.
+ *
+ * @param h
+ *   Hash table to iterate
+ * @param key
+ *   Output containing the key where current iterator
+ *   was pointing at.
```

Comment:
Improved sentence:
`Output containing the key at where the iterator is currently pointing.`